grails-app/migrations/0.9.x/changelog-2025-09-17-1435-add-column-order-item-reason-code.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+    <changeSet author="druchniewicz" id="170920251435-0">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="order_item" columnName="reason_code"/>
+            </not>
+        </preConditions>
+
+        <addColumn tableName="order_item">
+            <column name="reason_code" type="VARCHAR(255)"/>
```

Comment:
Let's call this discrepancy_reason_code. 

And I just realized that it would be a good idea to track this any time we split putaway tasks. For example, if there's not enough room in the primary location we can set the reason code = INSUFFICIENT_CAPACITY. If we return the split quantity into a hold location we can provide a reason code like DAMAGED or QUARANTINE.

Reply:
Make the change to the reason_code property name but don't change the required behavior in the mobile app unless it's easy to do. I created a comment in the ticket to track all of my feedback from the video and will create separate tickets for all of it.